### DISCUSSION TITLE
HasPlayer() 를 찾지못함

### DISCUSSION THREAD

**Discussion by smlee2952** [naverz/zepeto-studio-kor#957] [5. Q&A]
https://www.youtube.com/watch?v=H92Gd2G9DhM

위 유튜브를 보고 따라 진행하였으나, 아래 그림과 같은 에러가 발생하면서 테스트가 되지 않습니다.
빌드도 잘 되는데 실행만 하면 아래와 같은 에러가 발생합니다.

vscode 상 32번째 줄 HasPlayer() 를 찾을 수 없다고 Unity에서 에러를 출력합니다.
또한 가끔 vscode 상 46번째 줄 OnAddedLocalPlayer() 또한 찾을 수 없다고 에러를 출력합니다.

확인 부탁드립니다.

![image](https://user-images.githubusercontent.com/49216499/177156920-20f1aec7-cea2-4eab-be73-7e2ee80cb422.png)
![image](https://user-images.githubusercontent.com/49216499/177156787-9125fe12-82b5-4ceb-b928-6b6be0f8fd2d.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, https://github.com/naverz/zepeto-studio-kor/issues/958 와 동일 이슈입니다.
추후에 동일한 문제를 겪는 분들께서 참고 부탁 드립니다. 
감사합니다.
